docs/design/2022-08-31-placement-rules-in-SQL-works-with-tables-with-TiFlash-replica.md

```diff
@@ -0,0 +1,29 @@
+# Placement rules in SQL works with tables with TiFlash replica
+
+- Author: [Yunyan Hong](https://github.com/hongyunyan)
+
+## Introduction
+
+This RFC enables tables to support both setting placement rules and setting tiflash replicas.
+
+## Motivation or Background
+
+Currently, there is a conflict between setting tiflash replica and setting placement rules, i.e., if the table has a tiflash replica, it is not supported to set placement rules for this table. Similarly, if a table is set with some placement rules, it is not allowed to set tiflash replica for this table. The conflict between tiflash replicas and placement rules has a negative impact on user experience. Thus, this RFC will resolve this conflict and support setting tiflash replica and placement rules on tables at the same time.
+
+The conflict between tiflash replicas and placement rules setting is because both setting tiflash replicas and setting placement rules will generate placement configuration in PD. In the configuration generated by setting placement rules, ```group_index = 40, group_override = true```; while in the configuration generated by tiflash replica, ```group_index = 0```, ```group_override = false```. According to the rules in PD, if ```group_override``` is true , the configuration will override the configuration of other groups whose ```group_index``` is smaller than its own; if ```group_override``` is false, other configurations will not be overridden. Therefore, placement rules will overwrite the setting of tiflash replicas. In order to avoid this problem, the coexistence of placement rules and tiflash replicas was prohibited in the original development.
```

Comment:
Usually we use single backquote for inlined codes. The triple backquote is used for block level codes. See markdown syntax: https://www.freecodecamp.org/news/how-to-format-code-in-markdown/